jerry-core/ecma/base/ecma-globals.h

```diff
@@ -186,9 +186,8 @@ typedef enum
   ECMA_INTERNAL_PROPERTY_NATIVE_CODE, /**< native handler location descriptor */
   ECMA_INTERNAL_PROPERTY_NATIVE_HANDLE, /**< native handle associated with an object */
   ECMA_INTERNAL_PROPERTY_FREE_CALLBACK, /**< object's native free callback */
-  ECMA_INTERNAL_PROPERTY_PRIMITIVE_STRING_VALUE, /**< [[Primitive value]] for String objects */
-  ECMA_INTERNAL_PROPERTY_PRIMITIVE_NUMBER_VALUE, /**< [[Primitive value]] for Number objects */
-  ECMA_INTERNAL_PROPERTY_PRIMITIVE_BOOLEAN_VALUE, /**< [[Primitive value]] for Boolean objects */
+  ECMA_INTERNAL_PROPERTY_ECMA_VALUE, /**< ecma-value except object */
```

Comment:
Was the [[Primitive vaue]] part intentionally removed from the comment? I thought it referred to the standard (8.6.2 Object Internal Properties and Methods, Table 9 - Internal Properties Only Defined for Some Objects).